airbyte-integrations/connectors/source-serpstat/source_serpstat/spec.yaml

```diff
@@ -0,0 +1,87 @@
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  type: object
+  required:
+    - api_key
+  properties:
+    api_key:
+      type: string
+      title: API Key
+      airbyte_secret: true
+      order: 0
+      description: >-
+        Serpstat API key can be found here:
+        https://serpstat.com/users/profile/
+    domain:
+      type: string
+      order: 1
+      title: Domain
+      default: serpstat.com
+      description: The domain name to get data for (ex. serpstat.com)
+    page_size:
+      type: integer
+      order: 2
+      title: Page size
+      default: 10
+      description: >-
+        The number of data rows per page to be returned. Each data row can
+        contain multiple data points. The max value is 1000. Reducing the size
+        of the page will result in fewer API credits spent.

```

Comment:
Always use the max size the API allows. Remove the parameter and use direct the max size in the connector.